test/playlist-loader.test.js

```diff
@@ -28,6 +34,745 @@ QUnit.module('Playlist Loader', {
   }
 });
 
+QUnit.test('updateSegments copies over properties', function(assert) {
+  assert.deepEqual(
+    [
+      { uri: 'test-uri-0', startTime: 0, endTime: 10 },
+      {
+        uri: 'test-uri-1',
+        startTime: 10,
+        endTime: 20,
+        attributes: { BANDWIDTH: 99, height: 4 }
```

Comment:
You don't need to change these because whats really being tested is object copy and merging, which doesn't care about the content, but it was a little confusing to see a combination of segment and playlist objects being used to test `updateSegments`.